### PR TITLE
BIGTOP-3491. Add Ubuntu 20.04 support to the Bigtop toolchain.

### PR DESCRIPTION
**Pull request by sekikn**

### CODE REVIEW

**Comment by sekikn** [comment]
I tested that `sudo bigtop_toolchain/bin/puppetize.sh`, `./gradlew toolchain`, and (after relogin) `/gradlew zookeeper-pkg -Dbuildwithdeps=true` worked with this PR on Ubuntu 20.04.

File: bigtop_toolchain/manifests/packages.pp [hunk 1]
```diff
@@ -204,7 +204,6 @@
         "bison",
         "flex",
         "python-dev",
-        "python-pip",
```
Comment (iwasakims) [inline bigtop_toolchain/manifests/packages.pp:207]: Can we remove pip (for Python 2) from all distributions?

Comment (sekikn) [reply bigtop_toolchain/manifests/packages.pp:207]: Yes I think so. Python2 is still used in some components, but pip2 is not as far as I can see.

Comment (iwasakims) [reply bigtop_toolchain/manifests/packages.pp:207]: I think we should see the effect on Ubuntu/Debian first by committing this as is.

**Review by iwasakims — APPROVED** [review]
+1.

**Pull request merged**